process-manager.js

```diff
@@ -142,7 +140,10 @@ class ProcessManager {
 }
 
 ProcessManager.cache = processManagers;
-module.exports = ProcessManager;
+module.exports = {
+	ProcessManager: ProcessManager,
+	ProcessWrapper: ProcessWrapper,
```

Comment:
We'd normally call it `ProcessManager.ProcessWrapper`.

See `Users.Connection`, etc.